mkdocs.yml

```diff
@@ -1,6 +1,6 @@
 site_name: Flux | GitOps Toolkit
-site_description: A toolkit for assembling GitOps pipelines on Kubernetes
-site_author: The Flux project
+site_description: Open and extensible continuous delivery solution for Kubernetes
+site_author: The Flux authors
```

Comment:
This was done on purpose, see the commit message here: [`4319382` (#395)](https://github.com/fluxcd/flux2/pull/395/commits/4319382da3ff21528efdabe2362723af099a3b94)